data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2617,10 +2674,23 @@ spec:
                       OpenShift ingress port
                     type: string
                   ingressVIP:
-                    description: 'IngressVIP is the static IP on the nodes subnet
-                      that the apps port for openshift will be assigned Default: will
-                      be set to the 7 on the first entry in the machineNewtwork CIDR'
+                    description: 'DeprecatedIngressVIP is the static IP on the nodes
+                      subnet that the apps port for openshift will be assigned Default:
+                      will be set to the 7 on the first entry in the machineNewtwork
+                      CIDR Deprecated: Use IngressVIPs'
+                    format: ip
                     type: string
+                  ingressVIPs:
+                    description: IngressVIPs contains the VIP(s) on the nodes subnet
+                      that the apps port for openshift will be assigned. In dual stack
+                      clusters it contains an IPv4 and IPv6 address, otherwise only
```

Comment:
Are there default value(s) for IngressVIPs?

Reply:
Yeah they have default values on OpenStack. I updated the comment. In the code the default values are assigned e.g. here: 
https://github.com/openshift/installer/blob/e7c39f77ce018823a05caf962dc94493aa11ad87/pkg/types/openstack/defaults/platform.go#L45-L53